documentation/tutorials/sensitivity/PEcAn_sensitivity_tutorial_v1.0.Rmd

```diff
@@ -101,46 +107,40 @@ You should see a list of long numbers. Each of these is an ID. In the Files pane
 
 Let’s try to read the output from a single run id, as you did in the earlier tutorial. **read.output** is a PEcAn function to read in variables from any PEcAn compliant output files, it requires a runid, the directory, start year, end year, and an optional list of variables. Make sure you have RunDir variable set to the run number and the year to the year you ran.
 ```{r,eval=FALSE}
-arun <- read.output(runids[1],paste("~/output",RunDir,"out",runids[1],sep="/"),2006,2006,c("time","NEE"))
-plot(arun$time,arun$NEE)
+arun<-read.output()
+arun <- read.output(runids[1], paste("~/output", RunDir, "out", runids[1], sep="/"), start.year= 2004, end.year= 2004,"NEE", dataframe = TRUE)
+plot(arun$posix,arun$NEE)
 ```
 
 ####B. Now let’s bring in the actual observations
-Recall reading Ameriflux NEE in the earlier tutorial. This file was downloaded when we chose to get the drivers for SIPNET from Ameriflux. If it’s already in your workspace, you can skip this step. Otherwise, let's do it again.
-
-Navigate to the *~/output/dbfiles* folder. Read the Ameriflux NetCDF (.nc) file in  *~/output/dbfiles/Ameriflux_site-0-722/*  here *0-722* is the database site ID, a folder that was created when you selected the site and PEcAn downloaded the Ameriflux NetCDF (.nc) file from the Ameriflux website. The *0-722* may be a different number in your system depending on the site you chose and the database contents. Also note *US-NR1*, which refers to the Fluxnet site identifier for Niwot Ridge and the *2006* for the year.
+Recall reading Ameriflux NEE in the earlier tutorial. Read in an Ameriflux csv file, stored in the bety database. Here *722* is the database site ID which allows bety to assign the correct location of the data. The *722* may be a different number in your system depending on the site you chose and the database contents. Also note *US-NR1*, which refers to the Fluxnet site identifier for Niwot Ridge.
 ```{r,eval=FALSE}
-library(PEcAn.assim.batch)
-obs <- load.L2Ameriflux.cf("~/output/dbfiles/Ameriflux_site_0-772/US-NR1.2006.nc")
-names(obs)
-```
+File_Path<-"/fs/data3/tmccabe/AMF_US-NR1_BASE_HH_9-1.csv" #This is a path to where the observations are stored. 
+File_format<-PEcAn.DB::query.format.vars(bety = bety, format.id = 5000000002) #This matches the file with a premade "format" or a template that describes how the information in the file is organized
+site<-PEcAn.DB::query.site(site.id = 772,bety$con)  
 
-Change all bad values from -9999 to not a number (NA), and plot it to see if it make sense
-```{r,eval=FALSE}
-obs[obs==-9999]=NA
-plot(obs$NEE)
-```
+obs<-PEcAn.benchmark::load_data(data.path = File_Path, format= File_format, time.row = File_format$time.row,  site = site, start_year = 2004, end_year = 2004) 
 
```

Comment:
I'd add that these should be the exact same steps as the visualization tutorial, so you just need to reference that rather than explain much -- but you should make that reference more explicit since 'earlier tutorial' is vague and there's no guarantee that non Flux Course users will do the tutorials in the same order as the flux course students.